disnake/guild.py

```diff
@@ -177,7 +177,7 @@ class Guild(Hashable):
         The channel that denotes the AFK channel. ``None`` if it doesn't exist.
     id: :class:`int`
         The guild's ID.
-    owner_id: :class:`int`
+    owner_id: :class:`Optional[int]`
```

Comment:
```suggestion
    owner_id: Optional[:class:`int`]
```
`Optional` needs to wrap the `:class:`, otherwise intersphinx unfortunately [can't find it](https://github.com/DisnakeDev/disnake/actions/runs/12831038398/job/35780535385?pr=1273#step:4:355)